README.md

```diff
@@ -35,16 +37,16 @@ These are only server requirements. Each driver might have its own requirements.
 To install Appium 2 server using Node Package Manager (npm) run the following command:
 
 ```bash
-npm i --location=global appium
+npm i -g appium
```

Comment:
the -g argument has been deprecated in the recent NPM releases

Reply:
This change has since been reverted (as of npm 8.12.1), that is why I changed it back